runner/app/live/streamer/process.py

```diff
@@ -37,10 +37,12 @@ def __init__(self, pipeline_name: str):
         self.start_time = 0.0
 
     async def stop(self):
-        await asyncio.to_thread(self._stop_sync)
+        self._stop_sync()
```

Comment:
Can you add a TODO here to avoid calling synchronous stuff from an asynchronous context? It's ok for now but ideally we shouldn't be doing this